lms/djangoapps/grades/signals/handlers.py

```diff
@@ -105,8 +105,8 @@ def score_published_handler(sender, block, user, raw_earned, raw_possible, only_
                 )
 
     if update_score:
-        set_score(user.id, block.location, raw_earned, raw_possible)
-
+        module = set_score(user.id, block.location, raw_earned, raw_possible)
+        modified_time = module.modified.strftime("%y/%m/%d/%H/%M/%S/%f") if module else None
```

Comment:
Moving this to the signal handler that actually calls `recalculate_subsection_grade.apply_async`, since that's where we need this in string form.